test/int/adapter.test.ts

```diff
@@ -185,5 +185,38 @@ suite('Chrome Debug Adapter etc', () => {
             // force kill chrome here, as it will be left open by the debug adapter (same behavior as v1)
             killAllChrome();
         });
+
+
+        test('Should launch successfully on port 0', async () => {
+            // browser already launched to the default port, and navigated away from about:blank
+            const remoteDebuggingPort = 0;
+            await Promise.all([
+                dc.configurationSequence(),
+                dc.launch({ url: 'http://localhost:7890', timeout: 2000, webRoot: testProjectRoot, port: remoteDebuggingPort }),
+            ]);
+            // send some commands over CDP to verify the connection
+            const pauseResult = await dc.pauseRequest({threadId: 0});
```

Comment:
I think that an evaluate window.location.href is probably a more reliable way to test that you are indeed connected to Chrome.

Reply:
Not sure I understand this one. How would I check window.location.href, and why would that be more reliable?

Reply:
Talked offline. This is done